src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comment: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comment }) => (
+  <div className="CommentList">
+    {
+      comment.map((oneComment) => (
+        <ul>
+          <CommentInfo key={oneComment.id} info={oneComment} />
```

Comment:
Maybe you can rename variables, because in argument you get array of comments
```suggestion
export const CommentList: React.FC<Props> = ({ comments }) => (
  <div className="CommentList">
    {
      comment.map((oneComment) => (
        <ul>
          <CommentInfo key={oneComment.id} comment={oneComment} />
```